src/Microsoft.EntityFrameworkCore.AutoHistory/Extensions/DbContextExtensions.cs

```diff
@@ -20,17 +21,28 @@ public static class DbContextExtensions
         /// Ensures the automatic history.
         /// </summary>
         /// <param name="context">The context.</param>
-        public static void EnsureAutoHistory(this DbContext context)
+        /// <param name="useAddState">EntityStatus.Added will user too</param>
+        public static void EnsureAutoHistory(this DbContext context, bool useAddState = false)
```

Comment:
useAddState maybe rename to includeAddedState will be better?

Reply:
Ok. This is good too. Thanks